templates/articles/blocks/promoted_pages.html

```diff
@@ -1,47 +1,33 @@
-{% load wagtailimages_tags %}
-
-<h2>{{ value.heading }}</h2>
-<div class="row">
-    <ul class="card-group--list-style-none" data-container-name="promoted-pages" id="analytics-promoted-pages">
+{% load wagtailcore_tags wagtailimages_tags i18n %}
+<section class="promoted-pages body-container">
+    <h2 class="promoted-pages__heading">{% trans value.heading %}</h2>
+    <ul class="card-grid card-grid__trio card-grid--list promoted-pages__cards"
+        data-container-name="promoted-pages"
+        id="analytics-promoted-pages">
         {% for promoted_item in value.promoted_items %}
-
-        {% image promoted_item.teaser_image fill-288x172 as teaser_image_small %}
-        {% image promoted_item.teaser_image fill-328x196 as teaser_image_medium %}
-        {% image promoted_item.teaser_image fill-348x208 as teaser_image_large %}
-        {% image promoted_item.teaser_image fill-508x304 as teaser_image_extra_large %}
-
-        {% comment %}
-        Source copied from templates/includes/card-group-secondary-nav.html
-
-        Modified slightly to support data from a streamfield.
-
-        This is not intended to be production-ready code.
-
-        To discuss an approach to allow for reusability going forward.
-        {% endcomment %}
-
-        <li class="col-sm-12 col-md-6 col-lg-4 mb-3">
-            <div class="card-group-secondary-nav">
-                <a href="{{ promoted_item.url }}" class="card-group-secondary-nav__image-link" data-card-type="card-group-secondary-nav" data-card-title="{{ promoted_item.title }}" aria-hidden="true" tabindex="-1">
-                    <div class="card-group-secondary-nav__image">
-                        <picture>
-                            <source media="(max-width: 768px)" srcset="{{ teaser_image_extra_large.url }}">
-                            <source media="(max-width: 991px)" srcset="{{ teaser_image_medium.url }}">
-                            <source media="(max-width: 1199px)" srcset="{{ teaser_image_small.url }}">
-                            <source media="(min-width: 1200px)" srcset="{{ teaser_image_large.url }}">
-                            <img src="{{ teaser_image_small.url }}" alt="{{ image.title }}" class="card-group-secondary-nav__image-fallback">
-                        </picture>
-                    </div>
+            <li class="promoted-pages__card">
+                <a class="promoted-pages__card" href="{{ promoted_item.url }}">
```

Comment:
It's not a good idea to wrap entire card components in `<a>` tags, as for screen readers it means the entirety of the card content gets read aloud as a single string of words which they cannot pause or control. It's best to keep the link to the title only, which allows the user to tab through the other card content manually so they can read the image, title and description separately from each other using the arrow keys (or tab).

Reply:
I've updated the cards to follow the same approach we agreed for the rest of the cards for MVP.